pkgs/applications/virtualization/distrobox/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "distrobox";
+  version = "1.2.13";
+
+  src = fetchurl {
+    url = "https://github.com/89luca89/distrobox/archive/refs/tags/1.2.13.tar.gz";
+    sha256 = "00x6rrw1z2z2w1aw2l2v773n25rd25d8fmvz45i188gjp316kyq1";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  passthru.updateScript = ./update.sh;
```

Comment:
Maybe we can use `nix-update-script`?

Reply:
hm I dunno, I check it out and get back to you!